### ISSUE TITLE
Error in calculation of nucleotide number

### ISSUE THREAD

**Issue by mvences** [iTaxoTools/concatenator#66]
I have used Concatenator to obtain numbers of nucleotides in a fasta file, and found a bug in the calculation. It seems that for some of the counts, the algorithm has a bug and provides a number that is approximately double from what should be provided. I attach the example file here. The problems occur in the following reports:
- Total: Average number of nucleotides per sample / Minimum number of nucleotides per sample / Maximum number of nucleotides per sample
- Per sample: Total number of nucleotides (note that here, "Sequence length minimum" and "Sequence length maximum" are correct, and (since there is only one marker in the example file) both are the same; the "Total number of nucleotides" is double those numbers while it should be identical as well). 
- Total and Per marker: I think the same problem also applies to the "Total number of nucleotides" which probably should be half the value that is given in the report. 
- Note that in the report "Per input file", again, the values for "Sequence length minimum" and "Sequence length maximum" are correct. 
[16SConcatenatorTest.zip](https://github.com/iTaxoTools/concatenator/files/8733671/16SConcatenatorTest.zip)

**Comment by necrosovereign** [comment]
@StefanPatman
I tested the doubling issue with the total data in the backend and it gave correct numbers. It might be a problem with ConcatenatorGui

**Comment by necrosovereign** [comment]
This is weird. I tried it again with ConcatenatorGui and now the total report has the correct numbers. This problem seem to be caused by some particular combination of options.

**Comment by mvences** [comment]
Indeed. I have just tried it again with the compiled version and various input files, and it gave the correct numbers. This is very strange, I should have made screenshots and noted down the exact kind of steps I made, to better document the moment that the problem showed up. I will do some more tests when I have time, let's leave the issue open for now to make sure it is not forgotten. But it is a relief to know that the program in principle is performing correctly.
